Resfo Unformatted read fails in tests

This fails in ert-tests.
First appearance seems to have been during early hours of Saturdays bleeding build

```
FAILED unit_tests/shared/share/test_ecl_run.py::test_summary_block - assert False
 +  where False = isinstance([<resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8ddf40>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbbe0>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbca0>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbd00>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbd60>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbdc0>, ...], Summary)
 +    where [<resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8ddf40>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbbe0>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbca0>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbd00>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbd60>, <resfo._unformatted.read.UnformattedResArray object at 0x7f1b4b8cbdc0>, ...] = <bound method EclRun.summary_block of <ecl_run.EclRun object at 0x7f1b4b8ddac0>>()
 +      where <bound method EclRun.summary_block of <ecl_run.EclRun object at 0x7f1b4b8ddac0>> = <ecl_run.EclRun object at 0x7f1b4b8ddac0>.summary_block
FAILED unit_tests/shared/share/test_ecl_run_new_config.py::test_summary_block - assert False
 +  where False = isinstance([<resfo._unformatted.read.UnformattedResArray object at 0x7f4b5aad9400>, <resfo._unformatted.read.UnformattedResArray object at 0x7f4b3b03cd60>, <resfo._unformatted.read.UnformattedResArray object at 0x7f4b3b03cd00>, <resfo._unformatted.read.UnformattedResArray object at 0x7f4b3b03cfd0>, <resfo._unformatted.read.UnformattedResArray object at 0x7f4b3b0aa9a0>, <resfo._unformatted.read.UnformattedResArray object at 0x7f4b3b0aa6d0>, ...], Summary)
= 2 failed, 1900 passed, 74 skipped, 1 xfailed, 736 warnings in 1219.62s (0:20:19) =
```

--- a/src/ert/shared/share/ert/forward-models/res/script/ecl_run.py
+++ b/src/ert/shared/share/ert/forward-models/res/script/ecl_run.py
@@ -425,7 +425,9 @@ class EclRun:
             time.sleep(1)
 
             try:
-                ecl_sum = list(resfo.lazy_read(_find_unsmry(case)))
+                ecl_sum = [
+                    r.read_keyword() for r in resfo.lazy_read(_find_unsmry(case))
+                ]
             except Exception:
                 continue
 

--- a/tests/unit_tests/shared/share/test_ecl_run.py
+++ b/tests/unit_tests/shared/share/test_ecl_run.py
@@ -9,7 +9,6 @@ from textwrap import dedent
 
 import pytest
 import yaml
-from resdata.summary import Summary
 
 from tests.utils import SOURCE_DIR
 
@@ -475,7 +474,7 @@ def test_summary_block(init_ecl100_config, source_root):
     assert ret_value is None
 
     erun.runEclipse()
-    assert isinstance(erun.summary_block(), Summary)
+    assert erun.summary_block() is not None
 
 
 @pytest.mark.requires_eclipse

--- a/tests/unit_tests/shared/share/test_ecl_run_new_config.py
+++ b/tests/unit_tests/shared/share/test_ecl_run_new_config.py
@@ -8,7 +8,6 @@ from unittest import mock
 
 import pytest
 import yaml
-from resdata.summary import Summary
 
 from tests.utils import SOURCE_DIR
 
@@ -241,5 +240,4 @@ def test_summary_block(source_root):
     assert ret_value is None
 
     erun.runEclipse(eclrun_config=ecl_config.EclrunConfig(econfig, "2019.3"))
-    ecl_sum = erun.summary_block()
-    assert isinstance(ecl_sum, Summary)
+    assert erun.summary_block() is not None

